report/report.go

```diff
@@ -392,10 +392,10 @@ func (r *report) printSchemaListDiff(d *diff.SchemaListDiff) {
 		return
 	}
 
-	if d.Added > 0 {
+	if len(d.Added) > 0 {
 		r.print(d.Added, "schemas added")
```

Comment:
now that we know the names of the added schemas, let's print them

Reply:
I used this format, is that what you were thinking too?
```
	r.printConditional(len(d.Added) > 0, "Schemas added:", d.Added)
	r.printConditional(len(d.Deleted) > 0, "Schemas deleted:", d.Deleted)
```